backend/corpora/dataset_processing/process.py

```diff
@@ -254,24 +254,22 @@ def extract_metadata(filename):
 
     adata = scanpy.read_h5ad(filename, backed="r")
 
-    try:
-        raw_layer_name = [k for k, v in adata.uns["layer_descriptions"].items() if v == "raw"][0]
-    except (KeyError, IndexError):
-        raise RuntimeError("Raw layer not found in layer descriptions!")
-
-    if raw_layer_name == "X":
-        raw_layer = adata.X
-    elif raw_layer_name == "raw.X":
-        raw_layer = adata.raw.X
+    # TODO: Concern with respect to previous use of raising error when there is no raw layer.
```

Comment:
I'm not clear on what task this todo is referencing

Reply:
See the discussion [here](https://github.com/chanzuckerberg/single-cell-data-portal/pull/1358).

Reply:
Sorry I'm still a little confused. Is this a code change you are recommending? or just making a note of a particular issue?

Reply:
Previously the code used to raise an error if a raw layer wasn't present. The new code defaults to `AnnData.X` if the raw data isn't available. This is a significant change and was marked as a concern. That said, the product team seems onboard with the change, but I would like @pablo-gar to also confirm he's fine with it before we commit it. That said, I think adjustments can be made as part of a separate PR.